cornerstone_widget/cs_widget.py

```diff
@@ -149,32 +155,48 @@ def __init__(self, buttons_per_row=3,
                                      icon="play",
                                      button_style="success"
                                      )
+
         # We use the refresh button as a "start" button to
         # show the first image and then replace the on_click
         # handler after the first click
 
         def _first_click(button):
-            button.description = "Reset"
-            button.icon = "refresh"
-            button.button_style = ""
-            self._refresh_image()
+            # type: (widgets.Button) -> None
+            button.disabled = True
+
             button._click_handlers.callbacks.pop()
-            button.on_click(
-                lambda b: self._refresh_image()
-            )
+            self._refresh_image()
+            if show_reset:
+                button.description = "Reset"
+                button.icon = "refresh"
+                button.button_style = ""
+                button.on_click(
+                    lambda b: self._refresh_image()
+                )
+                button.disabled = False
+            else:
+                # TODO: is this a good way to delete a button?
```

Comment:
no idea. What is the use case for getting rid of it?

Reply:
cleaner interface without any buttons/toolbar